pkg/operator/operator_test.go

```diff
@@ -412,12 +412,12 @@ func TestRebalanceWorkloads(t *testing.T) {
 	)
 
 	for _, tc := range []struct {
-		name             string
-		pvs              []v1.PersistentVolume
-		pvcs             []v1.PersistentVolumeClaim
-		spreadByOperator bool
-		expectedPods     []string
-		expectedPVCs     []string
+		name               string
+		pvs                []v1.PersistentVolume
+		pvcs               []v1.PersistentVolumeClaim
+		balancedByOperator bool
+		expectedPods       []string
+		expectedPVCs       []string
```

Comment:
Do we need a test case when pods are placed on different nodes?

Reply:
I don't think so, in this function, it doesn't really matter whether the pods are scheduled on different nodes or not. Its only purpose is to cordon the node hosting the pod and deleting what needs to be deleted in order for the pod to be rescheduled somewhere else.